packages/form/addon/components/cf-field/input/date.js

```diff
@@ -1,27 +1,56 @@
 import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
 import { DateTime } from "luxon";
 
 export default class CfFieldInputDateComponent extends Component {
   @service intl;
 
+  @tracked flatpickrRef = null;
+
   get locale() {
     return this.intl.primaryLocale.split("-")[0];
   }
 
+  /*
+   * Extract the date format string for the current locale.
+   * For example en-us will return "m/d/Y" and de-ch will return "d.m.Y".
+   */
+  get dateFormat() {
+    const sample = this.intl.formatDate(
+      DateTime.fromISO("1970-12-31").toJSDate(),
+      {
+        day: "2-digit",
+        month: "2-digit",
+        year: "numeric",
+      }
+    );
+
+    // refer to flatpickr documentation for formatting options
+    return sample.replace(31, "d").replace(12, "m").replace(1970, "Y");
+  }
```

Comment:
Is there any other way to get the format string based on the current locale?

Reply:
I haven't found a better way, do you have an alternative?

Reply:
I did some research and I didn't find another way to do this generically either. Since we only really support three locales (de, fr, en) I would suggest simply defining the format in the `environment.js` and accessing the format through a constant mapping such as:

 ```suggestion
  get dateFormat() {
        return ENV.APP.LOCALE_DATE_FORMAT[this.locale];
  }
```
Should another locale be supported in the future we can add another format. What do you think?

Reply:
Put it into the env